Adding calgary-campinas dataset, and minor corrections.

emmetaobrien opened the pull request:
This PR does the following:

1) Adds the calgary-campinas CC359 dataset.
2) updates one file in refseq_Homo_sapiens to fix https://github.com/CONP-PCNO/conp-dataset/issues/474
3) corrects missing .datalad folder in two datasets from the Pavlidis lab to address https://github.com/CONP-PCNO/conp-dataset/issues/473

cmadjar reviewed (changes requested):
@emmetaobrien the README of the calgary-campinas dataset is displaying funny. The title seems weird with all those `=` and the Dataset Folder Structure should probably be in code format as it is not very readable.

Everything else looks fine :-)

emmetaobrien commented:
I have updated `calgary-campinas` with a new README.md  correcting the format issue, and also removed `projects/mm_neo_atlas` to address https://github.com/CONP-PCNO/conp-dataset/issues/481.

cmadjar reviewed (approved):
Thank you @emmetaobrien ! Looks good for me. Just waiting on Circle CI to finish the tests now.
